docs/modules/clients/pages/clc.adoc

```diff
@@ -0,0 +1,350 @@
+brew untap hazelcast/homebrew-hz            # optional
+----
+- Uninstall the script installation
+[source,bash,subs="attributes+"]
+----
+~/.local/share/hz-cli/bin/uninstall.sh
+----
+== Usage
+
+== Configuring HZC
+
+Hzc uses xref:go.adoc[hazelcast-go-client] under the hood to connect to cluster and to make operations.
+
+The default configuration is written to "~/.local/share/hz-cli/config.yaml" on your system on the first execution. You can modify this yaml file to change the configuration.
+
+You can also specify a custom config file with `--config/-c` flag. In this case, default file will not be considered.
+
+`hzc -c /some/custom/config.yaml`
+
+You can always override some fields of the configuration file with the following top level flags:
+
+- *--address/-a*: Addresses of the instances in the cluster (default is localhost:5701).
+- *--cloud-token*:  Hazelcast Cloud token
+- *--cluster-name*: name of the cluster to connect (default is dev)
+
+=== Configuring SSL
+
+You can use the following configuration file to enable SSL support:
+
+[source,yaml]
+----
+ssl:
+  servername: "HOSTNAME-FOR-SERVER"
+  # or: insecureskipverify: true
+hazelcast:
+  cluster:
+    security:
+      credentials:
+        username: "OPTIONAL USERNAME"
+        password: "OPTIONAL PASSWORD"
+    name: "CLUSTER-NAME"
+    network:
+    addresses:
+    - "localhost:5701"
+----
+
+Mutual authentication is also supported:
+[source,yaml]
+----
+ssl:
+    servername: "HOSTNAME-FOR-SERVER"
+    # insecureskipverify: true
+    capath: "/tmp/ca.pem"
+    certpath: "/tmp/cert.pem"
+    keypath: "/tmp/key.pem"
+    keypassword: "PASSWORD FOR THE KEY"
+hazelcast:
+  cluster:
+    security:
+      credentials:
+        username: "OPTIONAL USERNAME"
+        password: "OPTIONAL PASSWORD"
+    name: "CLUSTER-NAME"
+    network:
+      addresses:
+        - "localhost:5701"
+----
+
+Cloud SSL configuration:
+
+[source,yaml]
+----
+ssl:
+    capath: "/tmp/ca.pem"
+    certpath: "/tmp/cert.pem"
+    keypath: "/tmp/key.pem"
+    keypassword: "PASSWORD FOR THE KEY"
+hazelcast:
+  cluster:
+    name: "CLUSTER NAME"
+    cloud:
+      token: "HAZELCAST CLOUD TOKEN"
+      enabled: true
+----
+
+=== Modes
+
+*hzc* packs 3 usage modes to simplify different use cases.
+
+*Default (Non-interactive)*
+
+You can execute commands similar to any other command line tool. It can also be useful for scripting.
+[source,bash,subs="attributes+"]
+----
+hzc map --cluster-name cluster1 --name myMap put --key myKey --value myValue
+----
+
+*Interactive*
+
+This starts an interactive prompt, in which you can interactively navigate commands and explore possible subcommands and flags. It is also useful for manual tasks.
+
+image:ROOT:hzc-interactive-screenshot.png[HZC interactive]
+
+[source,bash,subs="attributes+"]
+----
+hzc
+----
+
+On this mode, you can also leverage the "use" command to avoid re-typing the distributed object name on each command. Note that map name "m1" is referenced only once.
```

Comment:
"In this mode" ?

Reply:
Thanks, fixed https://github.com/hazelcast/hz-docs/pull/462/commits/b3e94f20a0864ac722d6b633d8dc5a57e9758e7b